server/controllers/chartsController/helpers.js

```diff
@@ -421,3 +423,53 @@ export const intersectSubjectsFromFilters = (filters) => {
 
   return intersectedSubjects
 }
+
+export const sortTableRowDataBySite = (dataBySite) => {
+  return dataBySite
+    .map((site) => {
+      const { count, targetTotal } = site.totalsForStudy
+      site.counts[TOTAL_LABEL] = count
+      site.targets[TOTAL_LABEL] = targetTotal
+      return site
+    })
+    .sort(sortAllSitesBeforeTotalsSite)
+}
+
+const sortAllSitesBeforeTotalsSite = (siteA, siteB) => {
+  if (siteA.name === TOTALS_STUDY) return -1
+  if (siteB.name === TOTALS_STUDY) return -1
+}
+
+const formatGraphTableCellData = (studyCounts = 0, siteTarget) => {
```

Comment:
we should probably put the defaulted param last, or not default it.

Reply:
We have to default it sometimes it ends up undefined.
https://github.com/AMP-SCZ/dpdash/pull/287/commits/53c75c19215f81ea1f646fb5e98102019a21521f